.github/workflows/check-pull-request.yml

```diff
@@ -21,13 +21,10 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  CARGO_ARGS: ${{ github.ref == 'refs/heads/develop' && '--release' || '' }}
```

Comment:
I prefer that this is explicit in each run, and this also gave me the freedom to use debug for clippy.